tutorials/testing-coreutils.md

```diff
@@ -3,14 +3,14 @@ layout: default
 title: Testing Coreutils
 subtitle: Tutorial on How to Use KLEE to Test GNU Coreutils
 slug: tutorials
---- 
+---
 
 As a more detailed explanation of using KLEE, we will look at how we did our testing of [GNU Coreutils](http://www.gnu.org/software/coreutils/) using KLEE.
 
 This tutorial assumes that you have configured and built KLEE with `uclibc` and `POSIX` runtime support.
 
 These tests were done on a 32-bit Linux machine. On a 64-bit machine, we needed to also set the `LD_LIBRARY_PATH` environment variable:
```

Comment:
You actualy reran this on a 64-bit machine right? If so saying this was run on a 32-bit Linux machine is wrong.